sigilrecorder.lic

```diff
@@ -0,0 +1,290 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#sigilrecorder
+=end
+
+custom_require.call(%w[common common-arcana common-items common-travel drinfomon events spellmonitor equipmanager])
+
+class SigilRecorder
+	require 'yaml'
+
+	def initialize
+		#initialize arrays for supported towns, sigils, and seasons
+		@city_list = ['Crossing', 'Riverhaven', 'Shard']
+		@sigil_list = ['abolition','congruence','induction','permutation','rarefaction', 'antipode','ascension','clarification','decay','evolution','integration','metamorphosis','nurture','paradox','unity']
+		@season_list = ['summer', 'fall', 'winter', 'spring']
+
+		#pull data and settings and store the current region and season
+		@data = get_data('sigils')
+		@settings = get_settings
+		@region = get_region #parse Room.current.image for which region/city we're in
+		@season = get_season #make sure we're recording sigils to the correct season
+		
+		#generate an array of all the room numbers associated with the current room's Room.current.image value
+		region_list = region_room_list
+		#subtract known hunting spots and any other rooms in the YAML's nogo list to avoid combat and rooms with known mapping issues
+		walk_list = generate_nogo_list(region_list)
+		
+		#debug info
+		DRC.message("Region detected: #{@region}")
+		DRC.message("Season detected: #{@season.capitalize}")
+
+		#perform any outdoorsmanship buffs as this is the skill that governs sigil hunting
+		DRC.message("Checking for \"outdoors\" waggle set...")
```

Comment:
💡  As done on line 46, you can use the single quote here to not have to escape the double-quotes within double-quotes
```suggestion
		DRC.message("Checking for 'outdoors' waggle set...")
```

Reply:
Done!